SonarQube.Client/ISonarQubeService.cs

```diff
@@ -127,8 +127,8 @@ public interface ISonarQubeService
         Task<ServerExclusions> GetServerExclusions(string projectKey, CancellationToken token);
 
         /// <summary>
-        /// Creates a new <see cref="IServerSentEventsSession"/> for the given <see cref="projectKey"/>
+        /// Creates a new <see cref="ISSEStream"/> for the given <see cref="projectKey"/>
         /// </summary>
-        Task<IServerSentEventsSession> CreateServerSentEventsSession(string projectKey, CancellationToken token);
+        Task<ISSEStream> CreateServerSentEventsStream(string projectKey, CancellationToken token);
```

Comment:
CreateSSEStream? Or would that be too unclear?

Reply:
Hmm maybe from a readability point of view helps to spell out what SSE means? I don't care too much either way.